src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -626,7 +626,21 @@ private void LaunchOnlineHelp(Uri uriToLaunch)
                 browserProcess.StartInfo.Arguments = uriToLaunch.OriginalString;
                 browserProcess.Start();
 #elif CORECLR
-                throw new PlatformNotSupportedException();
+                if (Platform.IsNanoServer || Platform.IsIoT)
+                {
+                    throw new PlatformNotSupportedException();
+                }
+
+                browserProcess.StartInfo.FileName = "cmd.exe";
+                // start is very picky: the "optional" TITLE as the first argument should always be included, otherwise it can silently fail.
+                // also, we need "" around the uriToLaunch.OriginalString in case that there is a space in the HelpUri.
+                browserProcess.StartInfo.Arguments = string.Format(@"/c ""start /b """" ""{0}""""", uriToLaunch.OriginalString);
+                browserProcess.Start();
+                browserProcess.WaitForExit();
+                if (browserProcess.ExitCode != 0)
+                {
+                    throw new InvalidOperationException(browserProcess.StandardError.ReadToEnd());
```

Comment:
Meanwhile, the semantic would be different, but I think it's an acceptable deviation.